lab-brandon/ciphers/CaesarShiftCipher.java

```diff
@@ -0,0 +1,41 @@
+package ciphers;
+
+import java.util.Arrays;
+
+public class CaesarShiftCipher extends Cipher {
+
+    public static String encrypt (String payload, int shift) {
+        System.out.println(payload);
+        char[] arr = payload.toCharArray();
+        int ascii;
```

Comment:
You don't need to use ascii here or really worry about it at all for this problem